### DISCUSSION TITLE
When will it be published in Chrome Extensions?

### DISCUSSION THREAD

**Discussion by DougNeo** [ErnaneJ/tradingworks-plus#23] [Q&A] [answered]
Because is more easy for refresh the extensions when have a new release, do you have any predictions for this?

**Comment by ErnaneJ** [comment, marked as answer]
Maybe this will take a while. The chrome extension store requires an upfront payment for the developer license. I haven't thought about it yet but the expectation is that it will be published in the near future.

I'm just waiting for it to turn out the way it was planned as it's still in development. But when it's really ready, I'll publish it.

If you want to speed up this process and want to help with that, in addition to contributing to the repository, you can sponsor me.

Go [here](https://github.com/sponsors/ErnaneJ) and learn more about it.

Thanks!
